### DISCUSSION TITLE
Docs: avoid including prompt in fenced code blocks

### DISCUSSION THREAD

**Discussion by gdvalderrama** [fastapi/fastapi#12625] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
$ pip install something
```
```

Example from docs: https://github.com/fastapi/fastapi/blob/0.115.3/docs/en/docs/tutorial/request-files.md?plain=1#L12
```

### Description

In the docs, fenced code sometimes includes the prompt.
Because of this, when using the copy, the command copied includes an undesired prompt.

To avoid this, it would be best to not include the prompt.

So instead of

```console
$ pip install something
```

the code block would be

```console
pip install something
```

### Operating System

macOS

### FastAPI Version

0.115.3

### Pydantic Version

2.9.2

### Python Version

3.13.0

**Comment by Kfir-G** [comment]
@gdvalderrama A dollar sign (`$`) at the start of a line in a code block typically indicates it's a command to be run in a terminal. This is a common and recommended practice in technical documentation to clarify that the text is meant as input, not output. You can see more details in the [WordPress documentation style guide on command-line syntax](https://make.wordpress.org/docs/style-guide/developer-content/command-line-syntax/#command-prompt).

**Reply by gdvalderrama** [reply]
> A dollar sign ($) at the start of a line in a code block typically indicates it's a command to be run in a terminal.

Yes, this is common practice.
However, I think combining it with being able to copy the command including the `$` sign is counterintuitive.

I believe it should either not contain the `$` or not have the copy option.

**Comment by YuriiMotov** [comment]
This seems to be fixed. Works fine now
